login.html

```diff
@@ -1,46 +1,42 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="login.css">
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="login.css" />
     <title>login</title>
-</head>
-<body>
+  </head>
+  <body>
     <div class="logo_box">
-        <img class="logo" src="images/login/logo.png">
-        <div class="login_box">
-            <div id="email">
-                <span class="email">이메일</span>
-                <h2 class="email_input"></h2>
-            </div>
-            <div id="password">
-                <span class="password">비밀번호</span>
-                <h2 class="password_input"></h2>
-            </div>
-            <h3 class="login_button">
-                로그인
-            </h3>
-            <div id="easy_login">
-                <div class="easy_login_box">
-                    <h3 class="easy_login">간편 로그인하기</h3>   
-                    <div class="login_sns">
-                        <a href="https://www.google.com/"
-                            target="_blank">
-                            <img class="login_google" src="images/login/google.png">
-                        </a>
-                            <a href="https://www.kakaocorp.com/page/"
-                            target="_blank">
-                            <img class="login_kakao" src="images/login/kakao.png">
-                        </a>
-                    </div>
-                </div>
-            </div>
-            <div class="signup">
-                <p class="signup_ask">판다마켓이 처음이신가요?</p>
-                <a class="signup_link" href="signup.html">회원가입</a>
+      <img class="logo" src="images/login/logo.png" />
+      <div class="login_box">
+        <div id="email_password">
+          <span class="email_password">이메일</span>
+          <div class="email_password_input"></div>
+        </div>
+        <div id="email_password">
+          <span class="email_password">비밀번호</span>
+          <div class="email_password_input"></div>
+        </div>
+        <button class="login_button">로그인</button>
```

Comment:
💬 여담
button 태그의 type 속성을 명시하지 않으면 submit을 기본값으로 가지게 됩니다~
이러한 type 속성은 button 태그에 중요한 요소이므로 명시적으로 작성해주는 것을 추천드려요.

```suggestion
        <button type="submit" class="login_button">로그인</button>
```

+type이 submit인 button의 경우 form에서 제출 이벤트가 발생하면, 해당 요소의 onclick 이벤트가 실행되게 됩니다.
이를 더 풀어서 말하자면 form 태그안의 input, button 과 같은 상호작용 요소에서 submit 이벤트, enter key 입력이 발생하면 해당 form 에서의 submit 이벤트가 실행된다는 것입니다.
자세한 것은 추후 학습 하시면서 배우실 것이니 우선은 button 태그의 type 속성을 명시적으로 작성하는 것이 좋다는 것만 기억해주시면 좋겠습니다~

```html
// case1: form에 onsubmit 이벤트 명시
<form class="login-form" onsubmit="login">
  <input /> // input에서 enter 키 입력시 onsubmit 이벤트 실행
  <button type="submit">login</button> // 해당 버튼 클릭 시 onsubmit 이벤트 실행
</form>

// case2: submit 버튼에 onclick 이벤트 명시
<form class="login-form">
  <input /> // input에서 enter 키 입력시 button의 onclick 이벤트 실행
  <button type="submit" onclick="login">login</button>
</form>
```

[mdn button](https://developer.mozilla.org/ko/docs/Web/HTML/Element/button#type)